ledger/src/blockstore_metrics.rs

```diff
@@ -1,12 +1,12 @@
 use {
     crate::blockstore_db::{
-        columns, BlockstoreCompressionType, BlockstoreRocksDbColumnFamilyMetrics,
-        LedgerColumnOptions, ShredStorageType,
+        columns, BlockstoreCompressionType, LedgerColumnOptions, ShredStorageType,
     },
```

Comment:
I didn't think about this initially, but glad the compiler didn't yell at us here.
- `blockstore_metrics` pulls something in from `blockstore_db`
- `blockstore_db` pulls something in from `blockstore_metrics`

Reply:
Yep, I did notice this circular dependency.  If this is something not allowed, then we will need to move `ColumnMetric` to somewhere else.